setup.py

```diff
@@ -62,4 +69,18 @@
     install_requires=install_requires,
     extras_require=extras_require,
     entry_points={"console_scripts": ["atldld = atldld.cli:root"]},
+    description=description,
+    long_description=long_description,
+    license="LGPLv3",
+    classifiers=[
+        "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
+        "Operating System :: Unix",
```

Comment:
In Search we have `"Operating System :: MacOS",` maybe we can add it here?

Reply:
Done in 7c4f1f1 :)